### DISCUSSION TITLE
How to default IPv4 in node 17+ with the host as localhost?

### DISCUSSION THREAD

**Discussion by Vedanshh** [nodejs/help#4731] [Q&A]
### Version

node 17 and above

### What steps will reproduce the bug?

Create an express server and use the host as localhost.

### How often does it reproduce? Is there a required condition?

Every time for node 17 and above

### What is the expected behavior? Why is that the expected behavior?

In node version 16 and lower the default address was ipv4 but that's not the case with node 17 and above

### What do you see instead?

![IP issue](https://github.com/nodejs/node/assets/57327898/40011d0b-e8f1-4937-85b3-827ccab20d41)

I have tried a few solutions. It works if we mention the localhost IP (127.0.0.1) in the host. But how can we achieve the same with simply "localhost" as the hostname?

**Comment by Antonius-S** [comment]
https://nodejs.org/docs/latest-v19.x/api/dns.html#dnssetdefaultresultorderorder
